spr/spr.go

```diff
@@ -236,11 +237,21 @@ func (spr *StakingPriceRecord) CreateSPREntry() (*factom.Entry, error) {
 	if err != nil {
 		return nil, err
 	}
-	e.ExtIDs = [][]byte{{spr.Version}, rcd, spr.GetHash()}
+
 	e.Content, err = spr.SafeMarshal()
 	if err != nil {
 		return nil, err
 	}
+
+	signature, err := factom.SignData(spr.CoinbaseAddress, e.Content)
+	if err != nil {
+		return nil, err
```

Comment:
I see ` Method not found` error in this code line. @aktary

Reply:
Resolved by using the latest alpine factom-walletd docker image.  docker-compose.yml updated.